src/App.js

```diff
@@ -1,12 +1,46 @@
-import React from 'react';
+import React, {useState} from "react";
+import Button from "./components/Button";
+import GetPokemons from "./components/getPokemons";
 import './App.css';
+import Pokemon from "./assets/International_Pokémon_logo.svg.png"
 
 function App() {
+    const [offset, setOffset] = useState(0);
+
   return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
+      <>
+        <img className="pokemon-logo" src={ Pokemon }  alt="pokemon logo"/>
+        <div className="button-wrapper" >
+            <Button
+                name="vorige"
+                onClick={() => setOffset(offset - 20)}
+                disabled = { offset <= 0 && true }
```

Comment:
`&& true` is in deze overbodig
```suggestion
                disabled = { offset <= 0 }
```